packages/storybook/config/manager-head.html

```diff
@@ -48,11 +50,12 @@
     display: none;
   }
 
+  
   .os-content .sidebar-subheading {
     /* Title Small */
     letter-spacing: 0.02px;
     font-weight: 600;
-    color: #CBD0E0; /* n-6 */
+    color: var(--inovex-elements-n-3);
```

Comment:
Why did you pick n-3 over the previous n-6? It's quite a light contrast to the background. I think n-6 would be better, perhaps even higher - for accessibility and general readability.

n-3:
<img width="76" alt="image" src="https://github.com/inovex/elements/assets/48513535/d1698eb5-ae24-4872-acbd-d6125c0f2b32">

n-6:
<img width="49" alt="image" src="https://github.com/inovex/elements/assets/48513535/492b11c3-c457-4e60-9b2c-0097743aa89c">